Sources/Sentry/SentryTracer.m

```diff
@@ -642,41 +643,50 @@ - (nullable SentryAppStartMeasurement *)getAppStartMeasurement
         return @[];
     }
 
+    NSMutableArray<SentrySpan *> *appStartSpans = [NSMutableArray new];
```

Comment:
l: the idiomatic way to create a new array is:
```suggestion
    NSMutableArray<SentrySpan *> *appStartSpans = [NSMutableArray array];
```

Reply:
Any practical differences? Otherwise, I suggest sticking with the first approach, for me it's friendlier as it's the same for all classes.

Reply:
I built a habit of not using `+[new]` in pre-ARC code, but even with ARC now I don't use it as it can lead to weirdness with subclasses not calling designated initializers, like `NSManagedObject`. Whether or not `new` is used, we all need to be aware of the designated init for a given class, so it doesn't matter much what we end up committing, I guess. I just like not having the extra indirection, or having to double check every time I see a `new` whether or not `init` is OK to use for that class. `new` can also hide issues introduced in later API: https://blog.cristik.com/2018/02/objective-c-new-is-dangerous-and-old-avoid-it-at-all-costs/

Reply:
Didn't know that, thanks.

Reply:
I created an issue to create a linter for that https://github.com/getsentry/sentry-cocoa/issues/2126